src/themes/grayscale.js

```diff
@@ -0,0 +1,145 @@
+export default {
+  line: {
+    data: {
+      stroke: "#252525",
+      strokeWidth: 2,
+      fill: "none",
+      opacity: 1
+    },
+    labels: {
+      padding: 5,
+      fontFamily: "Poppins, Helvetica-Neue, sans-serif",
+      letterSpacing: 0.2,
+      fontSize: 10,
+      strokeWidth: 0,
+      stroke: "transparent",
+      textAnchor: "start"
+    },
+    parent: {
```

Comment:
I am not sure I understand the question. :( Are you asking if there needs to be any styles applied to the `<svg>` element? I have not thought about the implementation details—whatever you like! 

I do have a tangential question: if someone is using a custom `svg` wrapper with `standalone={false}`, would the default theme still apply? If yes, I suppose it’s best to avoid adding styles to the `<svg>` element. If no, make sure it fails in a way that’s still visually pleasing?